### PR TITLE
Feat playlist

### PR DESCRIPTION
**Pull request by amartya-rs**
In this PR: 
- added playlist features

Files to check: [review me](https://github.com/amartya-rs/hhTube/pull/7/files/673dfed2641a1533600c6ffdbc3ef76ddffe9a2a)

Preview: https://deploy-preview-7--hh-tube.netlify.app/

### CODE REVIEW

**Review by archit7448 — APPROVED** [review]
code looks good to merge. 
suggestion you can add something to give an indication of adding video in the playlist. Maybe you add  toast or something

File: src/components/modal/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      <div className={`modal-container ${!modalIsOpen ? "hide-modal" : ""}`}>
+         <div className="modal">
+            {pathname === "/playlist" ? (
+               <h5>Create new playlist</h5>
+            ) : (
+               <h5>Save to...</h5>
+            )}
+            {pathname === "/playlist" ? (
+               ""
+            ) : (
```
Comment (itsmebipul00) [inline src/components/modal/Modal.jsx]: Suggestion: the ternary operator could be sortened

**Review by itsmebipul00 — APPROVED** [review]
Awesome work!! good to merge

File: src/components/modal/Modal.jsx (hunk 1 above)
Comment (amartya-rs) [reply src/components/modal/Modal.jsx]: Done.

**Pull request merged**